### ISSUE TITLE
Possible IE11 bug with maximumFractionDigits

### ISSUE THREAD

**Issue by CWolfs** [sebastian-software/vue-locale#1]
I'm seeing a possible bug regarding IE11 and _maximumFractionDigits_ size linked [here](https://github.com/sebastian-software/vue-locale/blob/master/src/VueLocale.js#L36). 

When on IE11 and using _format-percent_ filter, the default max fraction digits of 20 seems to cause an error in intl-format-cache memoizer.s linked [here](https://github.com/yahoo/intl-format-cache/blob/master/src/memoizer.js#L24) with the error message 'Option value is outside of valid range'. Changing the _maximumFractionDigit_ in the library as a local test to something like '10' allows the code to work in IE11.

A workaround I've found is passing in the additional parameter for the _fractionDigits_ in the [format-percent filter](https://github.com/sebastian-software/vue-locale/blob/master/src/VueLocale.js#L245), however, since this sets the minimum and maximum decimal count to be this number it limits the flexibility of the format-percent filter.

Are you able to confirm this issue? I'm using vue-locale 0.3.2. Maybe you could provide two parameters of _minFraction_ and _maxFraction_ on the _format-number_ filter instead of just the one of _fractionDigits_ parameter? Or consider lowering the default _maximumFractionDigits_ size to work with IE11? 

Thanks!

**Comment by fastner** [comment]
We hit the same error. IE11 crashes with a default max fraction digits of 20.

**Comment by fastner** [comment]
https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat says teh default is Math.max(minimumFractionDigits, 3). @swernerx why is this 20?

**Comment by swernerx** [comment]
I'll have a look at the issue today.

20 is not actually some magic number... This is defined somewhere in the standard...

**Comment by swernerx** [comment]
From the article @fastner linked:

"minimumFractionDigits: The minimum number of fraction digits to use. Possible values are from 0 to 20"
"maximumFractionDigits: The maximum number of fraction digits to use. Possible values are from 0 to 20"

**Comment by swernerx** [comment]
Same info from the MSDN docs:

"minimumFractionDigits: Specifies the minimum number of fractional digits to be used. 0-20"
"maximumFractionDigits: This value can range from minimumFractionDigits to 20."

Via: https://msdn.microsoft.com/en-us//library/dn342819(v=vs.94).aspx

**Comment by swernerx** [comment]
I think generally 10 digits should be more than enough for 99% of cases, but it might make sense to figure out where it crashes...

**Comment by swernerx** [comment]
I just did the following in IE11 in BrowserStack:

```
new Intl.NumberFormat("en-US", {minimumFractionDigits: 2, maximumrFractionDigits:20}).format(100) => 100.00
```

So it generally seems to work with `20`.

**Comment by swernerx** [comment]
When trying it with `21` I got the following error message:

```
Option value '21' for 'maximumFractionDigits' is outside of valid range. Expected: [0 - 20]
```

**Comment by swernerx** [comment]
The error seems to be more related to the `percent` formatting...

I see the same error:

`Option value is outside of valid range`

when doing: 

```
new Intl.NumberFormat("en-US", {minimumFractionDigits: null, maximumFractionDigits:20, style: "percent"}).format(0.2039920323)
```

**Comment by swernerx** [comment]
The error seems be be done with values <= 18...

I figure IE somehow "thinks" that the space+percent symbol accounts to the maximum fraction digits e.g. 18+2 => okay... while 20+2 => crash

**Comment by CWolfs** [comment]
Good to see you're able to reproduce it and narrow down the issue.

**Comment by fastner** [comment]
Okay, that makes sense. Could you please fix this and release a new version? 18 digits should definitaly be enough :)

**Comment by swernerx** [comment]
Fixed with v0.4.0. Please update.
